packages/fxa-settings/src/components/SVGExample/index.tsx

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import FFLogo from './ff-logo.svg';
+
+export const SVGExample = () => (
+  <div>
+    <FFLogo data-testid="logo" />
+  </div>
```

Comment:
This could be a beautiful one liner:
```
export const SVGExample = () => <FFLogo data-testid="logo" />;
```

Will we remove this example when we have a "real" SVG to use in the project?

Reply:
We could. I'm fine with either. Doesn't hurt to test that the SVG loader is working.